bitcoin/src/blockdata/locktime/absolute.rs

```diff
@@ -454,21 +454,51 @@ impl Height {
     /// assert_eq!(lock_time.to_consensus_u32(), n_lock_time);
     #[inline]
     pub fn to_consensus_u32(self) -> u32 { self.0 }
-}
 
-impl_parse_str_from_int_fallible!(Height, u32, from_consensus, Error);
+    /*
+    fn parse<S: AsRef<str> + Into<String>>(s: S) -> Result<Self, ParseError> {
+        let n = s.as_ref().parse::<i64>().map_err(ParseError::invalid_int(s))?
+        let n = u32::try_from(n).map_err(|_| ParseError::Conversion(n))?;
+        Self::from_consensus(n).map_err(Into::into)
+    }
+    */
+}
```

Comment:
In 670c16b912e757ab6e657b65394a5a84505ac42e:

Did you mean to leave this commented out code here?

Reply:
Definitely not, thanks!